frontend/src/pages/breadcrumbs.tsx

```diff
@@ -0,0 +1,40 @@
+import BreadcrumbGroup from '@cloudscape-design/components/breadcrumb-group'
+import {BreadcrumbGroupProps} from '@cloudscape-design/components/breadcrumb-group/interfaces'
+import {CancelableEventHandler} from '@cloudscape-design/components/internal/events'
+
+type Page = 'clusters' | 'custom-images' | 'official-images' | 'users'
+
+const pageBreadcrumbItems: Record<Page, BreadcrumbGroupProps.Item> = {
+  clusters: {text: 'Clusters', href: '#clusters'},
+  'custom-images': {text: 'Custom Images', href: '#custom-images'},
+  'official-images': {text: 'Official Images', href: '#official-images'},
+  users: {text: 'Users', href: '#users'},
+}
+
+export function breadcrumbItemFromSlug(slug: Page): BreadcrumbGroupProps.Item {
+  return pageBreadcrumbItems[slug]
+}
+
+export function breadcrumbItem(text: string): BreadcrumbGroupProps.Item {
+  return {text, href: '#'}
+}
+
+const mainBreadcrumItem = {text: 'ParallelCluster Manager', href: '/clusters'}
```

Comment:
nitpick: I think you can route to `/`. There's a redirection to `/clusters` inside the router so, again, we don't replicate the routing behavior here.

Reply:
Thanks, done in next revision!